docs/quickstart.mdx

```diff
@@ -57,12 +62,57 @@ siloed from one another.
 That’s it! You can now launch your account! 🥳 Now that you have all the basics set up, you can connect Permit to a demo application and see your
 defined policy come to life.
 
-## 3. What's Next?
+## 3. Creating a basic policy
+
+Once we register with Permit and set-up our organization, now it's time to configure out first simple policy. We will be working with the
+simplest policy model called RBAC (Role-based Access Control).
+
+The idea of this simple guide is to show you how you create your first role, add a resource, and actions to that resource, and then
+assign the role to a user.
+
+### Creating a role
+
+The first step is to create a role for each user. In this example, we will create a role called `Admin`.
+
+<video controls autoPlay loop>
+  <source src={createRole} />
+</video>
+
+### Creating a resource and actions
+
+Once we created our role, the next step is to create a resource, and actions that we can perform on that resource. In this example
+we create a resource called a `Document` and utilize the basic pre-defined actions Permit provides (`crate`, `read`, `update` and `delete`),
+as well as assign our own bespoke actions: `publish`.
+
+<video controls autoPlay loop>
+  <source src={createResource} />
+</video>
+
+### Creating a user
+
+Now that we have a role, resource and some actions, we need to add a user into Permit, and assign him the role we have created.
+
+Let's add a user called `sam@permit.io` and assign him the role of an `Admin`.
+
+<video controls autoPlay loop>
+  <source src={createUser} />
+</video>
+
+### Configuring permissions in the Policy editor
+
+Finally, we need to configure our permissions in the Policy Editor, defining what actions `Admin` role can perform on the `Document`
+resource.
+
+<video controls autoPlay loop>
+  <source src={checkPolicies} />
+</video>
+
```

Comment:
I would add here a short statement/admonition that Permit is much more than RBAC/ABAC and this is just basic to see it in action.